src/blackbox_decode.c

```diff
@@ -492,14 +496,80 @@ int getMajorVersion(flightLog_t *log)
     return atoi(strtok(fcVersion, "."));
 }
 
+/**
+ * Get the firmware semantic version as a string (e.g., "4.3.0")
+ * Returns the version string or NULL if version is unknown/invalid
+ */
+const char* getSemver(flightLog_t *log)
+{
+    if (!log->private->fcVersion[0]) {
+        return NULL; //Unknown version
+    }
+    return log->private->fcVersion;
+}
+
+/**
+ * Compare firmware semantic version against a target version string
+ * Returns: true if firmware >= target, false if firmware < target or version unknown
+ */
+bool semverGte(const char *currentVersion, const char *targetVersion)
+{
+    if (!currentVersion || !targetVersion) {
+        return false; //Unknown version
+    }
+    
+    // Parse current version
+    char currentVersionCopy[30];
+    strcpy(currentVersionCopy, currentVersion);
+    
+    char *currentMajorStr = strtok(currentVersionCopy, ".");
+    char *currentMinorStr = strtok(NULL, ".");
+    char *currentPatchStr = strtok(NULL, ".");
+    
+    if (!currentMajorStr || !currentMinorStr || !currentPatchStr) {
+        return false; //Invalid current version format
+    }
+    
+    int currentMajor = atoi(currentMajorStr);
+    int currentMinor = atoi(currentMinorStr);
+    int currentPatch = atoi(currentPatchStr);
+    
+    // Parse target version
+    char targetVersionCopy[30];
+    strcpy(targetVersionCopy, targetVersion);
+    
+    char *targetMajorStr = strtok(targetVersionCopy, ".");
+    char *targetMinorStr = strtok(NULL, ".");
+    char *targetPatchStr = strtok(NULL, ".");
+    
+    if (!targetMajorStr || !targetMinorStr || !targetPatchStr) {
+        return false; //Invalid target version format
+    }
+    
+    int targetMajor = atoi(targetMajorStr);
+    int targetMinor = atoi(targetMinorStr);
+    int targetPatch = atoi(targetPatchStr);
+    
+    // Compare versions
+    if (currentMajor > targetMajor) return true;
+    if (currentMajor < targetMajor) return false;
+    
+    if (currentMinor > targetMinor) return true;
+    if (currentMinor < targetMinor) return false;
+    
+    if (currentPatch >= targetPatch) return true;
+    
+    return false;
+}
```

Comment:
Please put this in semver.c

- Do not think we should care about patch versions, RC1, etc ?
- Why not use getMajorVersion and add getMinorVersion ?

Full example implementation: https://github.com/h2non/semver.c/blob/master/semver.c

Reply:
- getSemver is not part of semver - depends on client implementation
- Easy solution IMO is to just use major and minor part
- Or go all the way and add semver library

Reply:
full semver implementation done.